modules/scanning-images.adoc

```diff
@@ -70,68 +69,85 @@ Scanner can check for vulnerabilities in images that use the following package f
 * dpkg
 * RPM
 
-[discrete]
+[id="supported-programming-languages_{context}"]
 == Supported programming languages
 
 Scanner can check for vulnerabilities in dependencies for the following programming languages:
 
+* Go (supported in Scanner V4 only)
 * Java
 * JavaScript
+* NodeJS (supported in Scanner V4 only)
 * Python
 * Ruby
 
-[discrete]
+[NOTE]
+====
+The vulnerability source for languages, link:https://osv.dev/list[Open Source Vulnerabilities (OSV)], provides GitHub Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities. OSV is used for some Scanner V4 results. The {product-title-short} Scanner V4 uses the OSV database available at OSV.dev under license.
+====
+
+[id="supported-runtimes-frameworks_{context}"]
 == Supported runtimes and frameworks
 
 Beginning from {product-title} 3.0.50 (Scanner version 2.5.0), Scanner identifies vulnerabilities in the following developer platforms:
 
 * .NET Core
-* ASP.NET Core
 
-[discrete]
+[id="supported-operating-systems_{context}"]
 == Supported operating systems
 
 The supported platforms listed in this section are the distributions in which Scanner identifies vulnerabilities, and it is different from the supported platforms on which you can install {product-title}.
 
 Scanner identifies vulnerabilities in images that contain the following Linux distributions:
 
 |===
-| Distribution | Version
+| Distribution | Version | Vulnerability database
```

Comment:
keep in mind Scanner V4 now updates its support matrix automatically for some of these OSes, so we may need to be sure to update this upon every single release. We should add some kind of reminder for the team to evaluate this each release. Is there a set of steps docs team usually follows? If so, can we add this to that list?

Reply:
Let's remove the Vulnerability database and instead link people to the list you have on the architecture page

Reply:
> keep in mind Scanner V4 now updates its support matrix automatically for some of these OSes, so we may need to be sure to update this upon every single release. We should add some kind of reminder for the team to evaluate this each release. Is there a set of steps docs team usually follows? If so, can we add this to that list?

The closest thing I can think of is creating a ticket for each release - which relies upon us remembering to do that, but, we don't really have a checklist for each release. I created one for 4.5: https://issues.redhat.com/browse/ROX-23197